github.py

```diff
@@ -1,8 +1,14 @@
 import json
 import requests
 import yaml
+import os
 
-token = raw_input('Paste your GitHub token? (see https://github.com/settings/tokens): ')
+envToken = os.environ['GITHUB_TOKEN']
+
+token = ''
+if envToken == '':
```

Comment:
and what if it's not empty? where you use it? maybe you wanna write some test.

Reply:
if the `envToken` is not empty, we'll use it as the github token.
my idea was that the env-var is the first one that will be checked by the script.
if i have time, i will write a test.

Reply:
yeah @paulvollmer I mean that the var is not referenced after this place.

Reply:
ah, got it. 
i changed it to `os.environ.get(...)` to not raise a `KeyError` and replaced the `envToken` variable with the `token`. i think we do not need one more variable.